.github/linters/c-sharp-lint.yml

```diff
@@ -0,0 +1,7 @@
+[*]
```

Comment:
This looks a lot like an `.editorconfig` file, do you think the action could use one instead?

Reply:
We already have one in ./src/.editorconfig, perhaps we could update it?